### ISSUE TITLE
[Feature] Add value for config-sync that describes current status 

### ISSUE THREAD

**Issue by seraphblade2010** [mbecker20/komodo#207]
I have been transitioning my homelab from UI defined to a written git resource sync and have come upon a missing value when defining a stack. Currently when a stack should be down or destroyed, there is no way to define this in the sync file.
 
 Is it possible to get a value that defines if a stack should be deployed, down or even destroyed?

**Comment by mbecker20** [comment]
Yeah I think you are right, the current solution of only `deploy = true` is not dynamic enough solution and I think a mistake to do it this way. I think it may be best to remove this `deploy = true` from the TOML and replace with something like:

- `ensure = "Running"`
- `ensure = "Stopped"`

It would be a breaking change to anyone using `deploy = true` though. It could also keep `deploy = true` just add `stopped = true`, but its a bit more awkward IMO, and less flexible.
